Jenkinsfile.download_all_logs

```diff
@@ -5,7 +5,7 @@ pipeline {
         string(name: 'REMOTE_SERVICE_URL', defaultValue: 'https://api.openshift.com', description: 'Service URL')
     }
 
-    triggers { cron('0 */2 * * *') }
+    triggers { cron('* * * * *') }
```

Comment:
Every minute? Isn't that a little bit too much?

Reply:
I don't think it should run every minute.

Reply:
1, it's not running every minute.
2. even if it will run with this cron it will run back to back as a constant sampler